src/builder/index.ts

```diff
@@ -153,19 +153,23 @@ export class Builder<
     }
     return this.globWithOptions(
       patterns,
-      ...[{dot: true}] as unknown as GlobParams<TGlobFunction>
+      ...([{ dot: true }] as unknown as GlobParams<TGlobFunction>)
     );
   }
 
   globWithOptions(patterns: string[]): Builder<TReturnType, TGlobFunction>;
-  globWithOptions(patterns: string[], ...options: GlobParams<TGlobFunction>): Builder<TReturnType, TGlobFunction>;
-  globWithOptions(patterns: string[], ...options: GlobParams<TGlobFunction>|[]) {
+  globWithOptions(
+    patterns: string[],
+    ...options: GlobParams<TGlobFunction>
+  ): Builder<TReturnType, TGlobFunction>;
```

Comment:
this would probably be easier to read with a new line between the method declarations
```suggestion

    globWithOptions(
    patterns: string[],
    ...options: GlobParams<TGlobFunction>
  ): Builder<TReturnType, TGlobFunction>;

```